types/ps-tree/ps-tree-tests.ts

```diff
@@ -0,0 +1,9 @@
+import * as ps_tree from "ps-tree";
```

Comment:
Since this is a callable entity, it isn't legal in ES2015 to import this with a namespace import like `import * as foo from 'foo'`, so could you please convert this to either

```ts
import foo = require('foo')
```

or the following with `esModuleInterop` turned on

```ts
import foo from 'foo';
```